clusters/app.ci/pr-reminder/pr-reminder-config.yaml

```diff
@@ -71,3 +71,40 @@ data:
         repos:
           - openshift/baremetal-runtimecfg
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
+          - jsafrane
+          - rbednar
+        teamNames:
+          - openshift-team-storage
+        repos:
+          - openshift/alibaba-cloud-csi-driver
+          - openshift/alibaba-disk-csi-driver-operator
+          - openshift/aws-ebs-csi-driver
+          - openshift/aws-ebs-csi-driver-operator
+          - openshift/aws-efs-csi-driver
+          - openshift/aws-efs-csi-driver-operator
+          - openshift/aws-efs-utils
+          - openshift/azure-disk-csi-driver
+          - openshift/azure-disk-csi-driver-operator
+          - openshift/azure-file-csi-driver
+          - openshift/azure-file-csi-driver-operator
+          - openshift/cloud-provider-openstack
+          - openshift/cluster-storage-operator
+          - openshift/csi-driver-manila-operator
+          - openshift/csi-external-attacher
+          - openshift/csi-external-provisioner
+          - openshift/csi-external-resizer
+          - openshift/csi-external-snapshotter
+          - openshift/enhancements
```

Comment:
Does this mean we get reminders for _any_ PR in openshift/enhancements? Or only from our team members?

Reply:
Correction, with
```
        teamNames:
          - openshift-team-storage
```
@openshift/openshift-team-storage must be tagged. I'd vote for `- storage` only

Reply:
Ack, removing `openshift-team-storage`.